src/modules/backups/models/device.model.ts

```diff
@@ -34,6 +35,19 @@ export class DeviceModel extends Model implements DeviceAttributes {
   @Column
   userId: number;
 
+  @AllowNull
+  @ForeignKey(() => FolderModel)
+  @Column({
+    type: DataType.UUID,
+    field: 'folder_uuid',
+  })
+  folderUuid: string;
```

Comment:
I assume this is required for the already existing ones, although is it better to do it this way or to use a fixed value for the already existing and not null in general?